.gitignore

```diff
@@ -20,6 +20,7 @@ testbin/*
 
 # editor and IDE paraphernalia
 .idea
+.vscode
```

Comment:
if you can share launch configuration, it might be very useful for others to run/debug without any extra configurations :)

Reply:
this config i'm ignoring isn't a run/debug config -- it looks like it's a by-product of installing a makefile plugin..